docs/program-setup/build-profiles-rules/debug_builds_cli.md

```diff
@@ -1,6 +1,6 @@
 <h1 id="debug-builds-cli">Debug builds</h1>
 
-After you've set up your [local debug toolchain](../program-setup/build-profiles-and-rules.html), you need firmware that includes program symbols (an `.elf` file). Because the Arm Mbed Online Compiler only produces binaries that omit the program symbols, you need to compile locally using [Arm Mbed CLI](../build-tools/mbed-cli-1.html).
+After you've set up your [local debug toolchain](../program-setup/build-profiles-and-rules.html), you need firmware that includes program symbols (an `.elf` file). You must compile locally using [Arm Mbed CLI](../build-tools/mbed-cli-1.html).
```

Comment:
@federicobozzini @JoeTheGuitarist See this page: https://os.mbed.com/docs/mbed-os/v6.15/program-setup/debug-builds-cli.html 
Does Mbed Studio produce binaries with program symbols? My understanding is that it is the case (but KSC does not include program symbols). Should we mention Mbed Studio/Keil Studio Cloud here at all?

Reply:
Yes, since Mbed Studio has a debugger it produces binaries with debug symbols. KSC does as well for debug sessions, although these debuggable binaries are not currently available in the output folder AFAIK.